message_definitions/v1.0/messages.xml

```diff
@@ -2837,6 +2850,17 @@
       <field name="ac_id" type="uint8"/>
     </message>
 
+    <message name="RC_6CH" id="74" link="broadcasted">
```

Comment:
the 6CH message has 7 channels?

Reply:
This is copied from the structure of the 5CH and 4CH messages. They respectively have 6 and 5 fields. So I am not sure whether this is wrong.

Reply:
Funny. Didn't notice that before. OK then.